<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>191</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Main Menu</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mapEdit">
            <property name="text">
             <string>Локальная сеть</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Локальная сеть</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Setings">
            <property name="text">
             <string>Настройки</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="close">
            <property name="text">
             <string>Выйти</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>190</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>191</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Menu 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_7">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>190</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="5">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>T3</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="T1">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>T2</string>
          </property>
         </widget>
        </item>
        <item row="3" column="7">
         <widget class="QLabel" name="Elect">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QLabel" name="Mech">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="13">
         <widget class="QLabel" name="Engine">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>T1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="12">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>D1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="22">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="0" column="10">
         <widget class="QLabel" name="T5">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="12">
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>Двиг.</string>
          </property>
         </widget>
        </item>
        <item row="0" column="13">
         <widget class="QLabel" name="D1">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="T2">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="17">
         <widget class="QLabel" name="D3">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="mainMenu">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>T4</string>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>T5</string>
          </property>
         </widget>
        </item>
        <item row="0" column="18">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>D4</string>
          </property>
         </widget>
        </item>
        <item row="0" column="15">
         <widget class="QLabel" name="D2">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="20">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>D5</string>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Элект.</string>
          </property>
         </widget>
        </item>
        <item row="0" column="16">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>D3</string>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QLabel" name="T4">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="10">
         <widget class="QLabel" name="Micro">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="21">
         <widget class="QLabel" name="D5">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="14">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>D2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="T3">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="Plate">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="8" colspan="2">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>Мик.Фаб.</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Graphics</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Лист.</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3" colspan="2">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Мех.</string>
          </property>
         </widget>
        </item>
        <item row="0" column="19">
         <widget class="QLabel" name="D4">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
